views/components/BarGraphTooltip/BarGraphTooltip.css

```diff
@@ -30,10 +30,18 @@
 }
 .BarGraphTooltip_label {
   grid-column-start: 1;
-  grid-column-end: 4;
+  grid-column-end: 5;
 }
 .BarGraphTooltip_values {
-  grid-column-start: 4;
-  grid-column-end: 5;
+  grid-column-start: 5;
+  grid-column-end: 6;
   text-align: right;
 }
+.BarGraphTooltip_divider {
+  grid-column-start: 1;
+  grid-column-end: 6;
+  height: 1.5px;
+  padding-bottom: 1px;
+  padding-top: 1px;
```

Comment:
I think the height is going to get messed up here because the padding top and bottom will increase it but it's restricted to 1.5px (which we should probably just round anyway). What do you think about removing the padding and setting the height to 3px or some other round number?

Reply:
[ce1d36e](https://github.com/AMP-SCZ/dpdash/pull/703/commits/ce1d36ed4023b15aba50cc47d2c0b91e243d8a54)